### DISCUSSION TITLE
How to send some package of HTTP requests simultaneously with a delay between ?

### DISCUSSION THREAD

**Discussion by killianweid** [symfony/symfony#47582] [Q&A]
I'm building an API with Symfony. During an action, the data coming from the front are website links and I use them to create and send asynchronous HTTP GET requests simultaneously (using of the Scrapestack API who scrapes these websites). But the fact is that the number of website links can be large and can be on the same domain. In order not to be blocked by a domain I would like to put a delay of 1sec between package of 10 requests sended simultaneously. Is it possible to do this with the PHP HTTP client Guzzle (https://github.com/guzzle/guzzle) ? Do I have to use Pool ?

Thanks for you help

**Comment by stof** [comment]
This is not the support channel for guzzle, which is not part of the Symfony project itself.

**Reply by killianweid** [reply]
@stof sorry but is it possible to do that in Symfony natively ?
